packages/plugin-prisma/src/field-builder.ts

```diff
@@ -145,6 +145,13 @@ fieldBuilderProto.prismaConnection = function prismaConnection<
           withUsageCheck: !!this.builder.options.prisma?.onUnusedQuery,
         });
 
+        const selection = info.fieldNodes[0];
+        const totalCountOnly =
+          selection.selectionSet?.selections.length === 1 &&
+          selection.selectionSet.selections.every(
+            (s) => 'name' in s && s.name.value === 'totalCount',
+          );
```

Comment:
I think this mostly makes sense, but I think rather than checking the field name, checking for an extension like pothosNoParent or something that indicates the field doesn't rely on the parent value, and then set that extension where the total count field is defined.

Reply:
@hayes thanks for the feedback, do you mind sharing an example of this in the codebase so I can use it as a reference?